plugins/generic/dublinCoreMeta/DublinCoreMetaPlugin.php

```diff
@@ -72,85 +71,94 @@ public function articleView($hookName, $args)
             return;
         }
 
+        $publication = $article->getCurrentPublication();
+        $publicationLocale = $publication->getData('locale');
+        $articleBestId = $publication->getData('urlPath') ?? $article->getId();
+
         $templateMgr = TemplateManager::getManager($request);
         $templateMgr->addHeader('dublinCoreSchema', '<link rel="schema.DC" href="http://purl.org/dc/elements/1.1/" />');
 
-        $i = 0;
-        if ($sponsors = $article->getSponsor(null)) {
-            foreach ($sponsors as $locale => $sponsor) {
-                $templateMgr->addHeader('dublinCoreSponsor' . $i++, '<meta name="DC.Contributor.Sponsor" xml:lang="' . htmlspecialchars(substr($locale, 0, 2)) . '" content="' . htmlspecialchars(strip_tags($sponsor)) . '"/>');
+        if ($supportingAgencies = $publication->getData('supportingAgencies')) {
+            foreach ($supportingAgencies as $locale => $localeSupportingAgencies) {
+                foreach ($localeSupportingAgencies as $i => $supportingAgency) {
+                    $templateMgr->addHeader('dublinCoreSponsor' . $locale . $i++, '<meta name="DC.Contributor.Sponsor" xml:lang="' . htmlspecialchars(substr($locale, 0, 2)) . '" content="' . htmlspecialchars($supportingAgency) . '"/>');
+                }
             }
         }
 
-        $i = 0;
-        if ($coverages = $article->getCoverage(null)) {
+        if ($coverages = $publication->getData('coverage')) {
             foreach ($coverages as $locale => $coverage) {
-                $templateMgr->addHeader('dublinCoreCoverage' . $i++, '<meta name="DC.Coverage" xml:lang="' . htmlspecialchars(substr($locale, 0, 2)) . '" content="' . htmlspecialchars(strip_tags($coverage)) . '"/>');
+                if (!empty($coverage)) {
```

Comment:
This probably isn't a case where it'll get us, but watch out for `empty` and values like `"0"` (string zero) or `0` (numeric zero). `empty('0')` and `empty(0)` both return `true`. Better to use `if (coverage != '')` (this is safe with nulls and numeric zeros too). (Also elsewhere)